app/src/test/java/org/simple/clinic/home/overdue/search/OverdueSearchEffectHandlerTest.kt

```diff
@@ -302,4 +302,18 @@ class OverdueSearchEffectHandlerTest {
     verifyNoMoreInteractions(uiActions)
     effectHandlerTestCase.assertNoOutgoingEvents()
   }
+
+  @Test
+  fun `when add all appointment ids effect is received, then all overdue appointments loaded`() {
+    // given
+    val allAppointmentIds = setOf(
+        UUID.fromString("9737ad9e-70f6-454e-9226-6eaa01c62645"))
+
+    // when
+    effectHandlerTestCase.dispatch(SelectAllAppointmentsIds(allAppointmentIds))
+
+    // then
+    verifyNoMoreInteractions(uiActions)
+    effectHandlerTestCase.assertNoOutgoingEvents()
```

Comment:
Seems like we are not testing anything in `//then` here. Maybe we should verify if  `addSelectedIDs` method is called or not in the selector? Also, when we add the then condition we should update the test name as well for the then condition.. we are not loading overdue appointments here, right?